assets/dev/js/editor/regions/panel/pages/elements/elements.js

```diff
@@ -142,6 +133,8 @@ PanelElementsLayoutView = Marionette.LayoutView.extend( {
 				title: categoryConfig.title,
 				icon: categoryConfig.icon,
 				defaultActive: categoryConfig.active,
+				sort: categoryConfig.sort,
+				dynamic: categoryConfig.dynamic,
```

Comment:
It's not self-documented. maybe `hideIfEmpty`

Reply:
> It's not self-documented. maybe `hideIfEmpty`

But it also expanded when widgets added..